src/components/dateTimePicker/index.js

```diff
@@ -0,0 +1,207 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {StyleSheet} from 'react-native';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import {Constants} from '../../helpers';
+import {Colors} from '../../style';
+import Assets from '../../assets';
+import {BaseComponent} from '../../commons';
+import {TextField} from '../inputs';
+import Dialog from '../dialog';
+import View from '../view';
+import Button from '../button';
+
+
+const MODES = {
+  DATE: 'date',
+  TIME: 'time'
+};
+
+/*eslint-disable*/
+/**
+ * @description: Date and Time Picker Component that wraps RNDateTimePicker for date and time modes.
+ * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/DateTimePickerScreen.js
+ * @important: DateTimePicker uses a native library. You MUST add and link the native library to both iOS and Android projects.
+ * @extends: @react-native-community/datetimepicker
+ * @extendslink: https://github.com/react-native-community/react-native-datetimepicker#react-native-datetimepicker
+ */
+/*eslint-enable*/
+
+class DateTimePicker extends BaseComponent {
+  static displayName = 'DateTimePicker';
+  static propTypes = {
+    ...TextField.propTypes,
+    /**
+     * The type of picker to display ('date' or 'time')
+     */
+    mode: PropTypes.oneOf(Object.values(MODES)),
+    /**
+     * The initial value to set the picker to. Defaults to device's date / time
+     */
+    value: PropTypes.instanceOf(Date),
```

Comment:
`value` is the name of the RNDateTimePicker prop

Reply:
The question is - should "fix" their API or stick to what they have... 
Their value act as an initial value ?

Reply:
Regarding the 'value' prop. Yes, their 'value' acts as an initial value, but I don't think we should fix their API. First, because it will be easier for those who want to migrate from their API to ours. Second, in our case, it also acts as the TextField 'value' prop so it's more intuitive in my opinion.